assets/sass/components/_applications.scss

```diff
@@ -0,0 +1,93 @@
+$statuses: (
+    'complete': #55ac3b,
+    'incomplete': #fd992b,
+    'empty': #7b7b7b,
+    'invalid': red
+);
+
+$statusColumnWidth: 160px;
+
+.app-summary__status {
```

Comment:
Mind scoping this within `app-summary`? Also I'd be inclined to call this `application-summary` as app reads a little meta software-y.